10-linux-modules-pre.hook

```diff
@@ -7,5 +7,5 @@ Target = usr/lib/modules/*/vmlinuz
 [Action]
 Description = Saving Linux kernel modules...
 When = PreTransaction
-Depends = rsync
-Exec = /bin/sh -c 'KVER="${KVER:-$(uname -r)}"; if test -e "/usr/lib/modules/${KVER}"; then rsync -AHXal --delete-after "/usr/lib/modules/${KVER}" /usr/lib/modules/backup/; fi'
+Depends = coreutils
+Exec = /bin/sh -xc 'KVER="${KVER:-$(uname -r)}"; if test -e "/usr/lib/modules/${KVER}"; then mkdir -p /usr/lib/modules/backup; cp -ral "/usr/lib/modules/${KVER}" /usr/lib/modules/backup; fi'
```

Comment:
With `cp -ral` you can actually drop the `-r`, as it is already included in the `-a` flag.